src/driver/close.rs

```diff
@@ -21,8 +21,8 @@ impl Op<Close> {
 impl Completable for Close {
     type Output = io::Result<()>;
 
-    fn complete(self, result: io::Result<u32>, _flags: u32) -> Self::Output {
-        let _ = result?;
+    fn complete(self, cqe: op::CqeResult) -> Self::Output {
+        let _ = cqe.result?;
```

Comment:
What does clippy say about this line? I've had clippy ask me to drop the 'let' when assigning to '_'.

Reply:
clippy is run in ci: its fine

Reply:
Also note, I haven't changed this